### ISSUE TITLE
Curse of dimensionality ?

### ISSUE THREAD

**Issue by WoodKieran** [parmoo/parmoo#70]
My problem, number of design variables = 600, solves within MOOP but gives me design suggestions which result in far from the global optimum (pre-determined).

Is this the curse of dimensionality or should I revisit my setup ?

Basics/ important parts of MOOP setup:

*********************
self.my_moop = libE_MOOP(TR_LBFGSB, hyperparams={})

self.my_moop.addSimulation(
    {
        'name': self.name,
        'm': self.m,
        'sim_func': self.Simulation,
        'search': LatinHypercube,
        'surrogate': LocalGaussRBF,
        'hyperparams': {'search_budget': 2000},
    }
)

for i in range(9):
    self.my_moop.addAcquisition({'acquisition': RandomConstraint,
                        'hyperparams': {}})
self.my_moop.addAcquisition({'acquisition': FixedWeights,
                        'hyperparams': {'weights': np.ones(2) / 2}})

self.my_moop.solve(50)

*********************

**Comment by thchang** [comment]
Hi @WoodKieran ,

There are some lines missing here, such as adding the objective functions, but I'll assume that is just information that you omitted.

I think the issue here is that your problem is way too high-dimensional and your budget is way too restrictive for such a large problem.

It looks like you are following our [Solving High-Dimensional Multiobjective Optimization Problems Tutorial](https://parmoo.readthedocs.io/en/latest/tutorials/local_method.html), which is a good start!

However, the complexity still grows *linearly* with the dimension.  As a rule of thumb:  you need iterations proportional to the number of design variables.

You have 600 design variables.  That means that every 600-ish iterations, you might expect to get the equivalent to 1-ish steps of gradient descent.  So for this problem, you would need to run ParMOO for thousands of iterations to get reasonable results.

The usage of multiple acquisition functions (you have 10) means that ParMOO will descend on 10 solutions in parallel and take 10 parallel evaluations per iterations.  It won't help you with reducing the total iteration requirement though...

I hope this is informative,

---Tyler
